climpred/bootstrap.py

```diff
@@ -345,6 +345,8 @@ def bootstrap_compute(
         if hist is None:  # PM path, use reference = control
             hist = reference
         uninit_hind = resample_uninit(hind, hist)
+        # Append lead time units from initialized forecast.
+        uninit_hind['lead'].attrs['units'] = hind['lead'].attrs['units']
```

Comment:
Can this go into resample_uninit?

Reply:
There's a lot of operations on the lead dimension in `bootstrap_uninitialized_ensemble` for instance so the attribute gets lost quickly. I think it's easier to cover here but I'll see if I can make it work on the others.